lib/payloadAction.ts

```diff
@@ -33,6 +35,29 @@ export const  handlePayloadActions = async (read: IRead,  modify: IModify, rid:
                 } else if (actionName === ActionIds.CLOSE_CHAT) {
                     await closeChat(modify, read, rid);
                 }
+                else if (actionName === ActionIds.SET_TIMEOUT) {
+                    let n = setTimeout(async () => {
```

Comment:
`setInterval`, `setTimeout`  is not supported in The RC Apps currently. For reference https://github.com/RocketChat/Apps.Dialogflow/issues/30

They do have Schedular API https://developer.rocket.chat/apps-development/fundamentals-of-apps/scheduler-api